package-lock.json

```diff
@@ -10897,32 +10962,25 @@
       "dev": true
     },
     "yargs": {
-      "version": "8.0.2",
-      "resolved": "https://registry.npmjs.org/yargs/-/yargs-8.0.2.tgz",
-      "integrity": "sha1-YpmpBVsc78lp/355wdkY3Osiw2A=",
+      "version": "10.0.3",
```

Comment:
there are a lot of irrelevant changes in package-lock.json. could you please remove them?